packages/jahia/migrations/migrate-to-v7.yaml

```
@@ -138,3 +139,13 @@ actions:
   addJelasticExposeFalse:
     - env.control.AddContainerEnvVars[sqldb]:
       vars: {"JELASTIC_EXPOSE": "false"}
+
+  updateIMFlavor:
+    - cmd[cp, proc]: |-
+        yum install -y https://rpms.remirepo.net/enterprise/remi-release-7.rpm
+        yum --enablerepo=remi install -y ImageMagick7
+        [ -f /usr/bin/magick ] || exit 1
+      user: root
+    - env.control.AddContainerEnvVars [cp, proc]:
```

Can add an idempotent check if it is already installed
```suggestion
    -  if ("${response.errOut}" == ""):
          env.control.AddContainerEnvVars [cp, proc]:
```